modules/p2/ding_entity_rating/ding_entity_rating.features.field_instance.inc

```diff
@@ -54,14 +54,14 @@ function ding_entity_rating_field_default_field_instances() {
         'label' => 'hidden',
         'module' => 'ding_entity_rating',
         'settings' => array(),
-        'type' => 'ding_entity_rating_action_default',
+        'type' => 'ding_entity_rating_action_ajax_load',
         'weight' => 2,
       ),
       'teaser' => array(
         'label' => 'hidden',
         'module' => 'ding_entity_rating',
         'settings' => array(),
-        'type' => 'ding_entity_rating_action_default',
+        'type' => 'ding_entity_rating_action_ajax_load',
```

Comment:
Only if a site manual has overridden the feature else it will read the setting from code after cache clear. But it's also part of the standard installation from DBC to do a `drush fra -y`.